libs/@guardian/atoms-rendering/src/YoutubeAtom.tsx

```diff
@@ -58,6 +61,10 @@ export const YoutubeAtom = ({
 	imaEnabled,
 	abTestParticipations,
 	videoCategory,
+	kicker,
+	pillar,
+	shouldPauseOutOfView,
```

Comment:
If this is undefined should we default to false here? Then it can become a required prop to `YoutubeAtomSticky.tsx`

Reply:
good point, I'll update this 👍